auslib/admin/views/rules.py

```diff
@@ -18,18 +18,25 @@ class RulesAPIView(AdminView):
 
     def get(self, **kwargs):
         rules = dbo.rules.getOrderedRules()
-        count = 0
-        _rules = []
-        for rule in rules:
-            _rules.append(dict(
-                (key, value)
-                for key, value in rule.items()
-            ))
-            count += 1
-        ret = {
-            "count": count,
-            "rules": _rules,
-        }
+        if request.args.get('channels_only'):
```

Comment:
You only have read access to this repository, so continue pushing to your own to update this pull request. You may want to create a branch in your repo, though.